packages/fast-components-styles-msft/src/utilities/colors.ts

```diff
@@ -153,11 +177,27 @@ export function ensureBrandNormal(config: DesignSystem): string {
 /**
  * Ensure the background color meets normal contrast ratios against a background color
  */
-export function ensuresBackgroundNormal(config: DesignSystem): string {
+function ensureBackgroundNormal(config: DesignSystem): string {
     const designSystem: DesignSystem = withDesignSystemDefaults(config);
     return ensureNormalContrast(
         designSystem.contrast,
         designSystem.backgroundColor,
         designSystem.foregroundColor
     );
 }
+
+/**
+ * Deprecated 3.1.0
```

Comment:
do you need to mark this with `@deprecated?`

Reply:
yea - the name is off. ensures vs ensure

Reply:
@nicholasrice sorry, I should've been more specific. I meant mark it using the actual jsdoc format so we get it in intellisense. Not blocking on anything, just curious if you missed that or intentionally left it out.

Reply:
ohh gotcha - yep, good catch.